student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+class Student < Person
+    attr_accessor :classroom
```

Comment:
- [ ] `classroom` is not modified. As such you don't need to set it as a `setter` and `getter`. You can delete this line of code.
- You can create the intialize method as below:
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```
- `Constructor extends parent's constructor by adding @classroom and a parameter for it.` You can [read more by clicking on this link](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars`)